README.md

```diff
@@ -395,19 +401,19 @@ prefix to the keyword: `_aflow_aflow_prototype_label_relax`.
 
 <div class="cell markdown">
 
-The repository
-[tachyontraveler/oqmd-optimade-tutorial](https://github.com/tachyontraveler/oqmd-optimade-tutorial)
-contains exercises pertaining to the [OQMD](https://oqmd.org) database,
-where you will construct a simple machine learning model for predicting
-band gaps from crystal structures. These exercises can also be accessed
+This interactive exercise explores the OQMD's OPTIMADE API, and
+demonstrates how you can train machine learning models on OPTIMADE data.
+The notebook is available at
+`./notebooks/exercise7-oqmd-optimade-tutorial` and can also be accessed
 online with
-[Colab](https://colab.research.google.com/github/tachyontraveler/oqmd-optimade-tutorial/blob/main/Workbook.ipynb)
+[Colab](https://colab.research.google.com/github/Materials-Consortia/optimade-tutorial-exercises/blob/main/notebooks/exercise7-oqmd-optimade-tutorial.ipynb)
 or
-[Binder](https://mybinder.org/v2/gh/tachyontraveler/oqmd-optimade-tutorial/HEAD?filepath=Workbook.ipynb).
+[Binder](https://mybinder.org/v2/gh/Materials-Consortia/optimade-tutorial-exercises/HEAD?filepath=notebooks/exercise7-oqmd-optimade-tutorial.ipynb)
```

Comment:
When I click on the binder button I get an error:

 404 : Not Found
You are requesting a page that does not exist!

Reply:
That link won't work until this PR has merged.

Reply:
Ah, yes you are right. When I clicked on the button I got redirected to a binder page and from the address is was not clear it loaded the page fromn the git hub.